conda/recipes/rapids-build-env/meta.yaml

```diff
@@ -109,6 +109,7 @@ requirements:
     - panel {{ panel_version }}
     - pickle5  # [py<38]
     - pkg-config
+    - pre_commit
```

Comment:
Yeah, I just chose this for consistency since we use `pre_commit` in all the conda files in rapids AFAIK, but that's something we can update incrementally I suppose.